### PR TITLE
Add on-keyboard and keyboard-shortcut element modifiers and promote keyboard-press component

### PR DESCRIPTION
**Pull request by lukemelia**
Depends on ember upgrade in https://github.com/briarsweetbriar/ember-keyboard/pull/110

New element modifiers are Ember 3.12+ only. In theory they should work on older Ember versions, but tests told a different story.

### CODE REVIEW

**Comment by emilkarl** [comment]
Will this be merged? :D

File: tests/dummy/app/templates/usage.hbs [hunk 1]
```diff
@@ -14,7 +14,16 @@ To declaratively attach an action to a keyboard event while a particular templat
 
 Note that the listener is active for the lifecycle of the template. When the template is unrendered, the listener is removed.
 
-Another option for templates is the `keyboard-shortcut` element modifier. This allows a keyboard event to simulate clicking the element the modifier is attached to. In the following example, we trigger a click on the button when the B key is pressed together with the Alt key:
+Another option for templates is the `on-keyboard` element modifier. This allows a keydown event to trigger an action. In the following example, we trigger an action when the input field is focused and the user presses enter:
+
+```hbs
+<input type='text'
+  oninput={{action (mut this.textFieldValue) value='target.value'}}
+  {{on-keyboard 'Enter' (action 'onEnterPressedInInput')}}
```
Comment (mattmcmanus) [inline tests/dummy/app/templates/usage.hbs:22]: Shouldn't this be `(fn this.onEnterPressedInInput)`?

Comment (lukemelia) [reply tests/dummy/app/templates/usage.hbs:22]: It depends on how it is defined in the component, controller, or route, which is not specified here.

Comment (mattmcmanus) [reply tests/dummy/app/templates/usage.hbs:22]: Clearly I have things to learn...

**Comment by jkeen** [comment]
Hey @lukemelia, you wanna add that PR I submitted on your fork? I completed some of the todos you made

**Comment by lukemelia** [comment]
Merged, @jkeen. Thanks for collaborating on this. Are you using the modifiers in production yet?

**Comment by jkeen** [comment]
@lukemelia I am! And it's been great. That `{{keyboard-shortcut}}` modifier is gold, makes it so easy.

**Pull request merged**